data/src/main/java/com/terning/data/dto/response/SearchScrapsResponseDto.kt

```diff
@@ -0,0 +1,28 @@
+package com.terning.data.dto.response
+
+import InternshipAnnouncement
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+
+@Serializable
+data class SearchScrapsResponseDto(
+    @SerialName("internshipAnnouncementId")
+    val internshipAnnouncementId: Long,
+    @SerialName("companyImage")
+    val companyImage: String,
+    @SerialName("title")
+    val title: String,
+) {
+    fun toSearchScrapEntity(): List<InternshipAnnouncement> {
+        return listOf(
+            InternshipAnnouncement(
+                announcementId = internshipAnnouncementId,
+                companyImage = companyImage,
+                title = title,
+            )
+        )
```

Comment:
```suggestion
    fun toSearchScrapEntity(): List<InternshipAnnouncement> =  listOf(
            InternshipAnnouncement(
                announcementId = internshipAnnouncementId,
                companyImage = companyImage,
                title = title,
            )
        )
```
요렇게 하면 코드가 예뻐질 것 같아요!